src/meep.hpp

```diff
@@ -1416,6 +1416,7 @@ class dft_ldos {
   std::complex<double> *F() const; // returns Fdft
   std::complex<double> *J() const; // returns Jdft
   std::vector<double> freq;
+  mutable double overall_scale;
```

Comment:
1. Make this a private member `saved_overall_scale` since it is read-only
2. Add an accessor function `double overall_scale() const { return saved_overall_scale; }`
3. Get rid of `mutable` and remove the `const` qualifier from `dft_ldos::ldos()`.